### PR TITLE
make code blocks use preformatted whitespace.

### PR DESCRIPTION
**Pull request by ratmice**
Signed-off-by: matt rice <ratmice@gmail.com>

If you look at https://docs.sel4.systems/GettingStarted 
the code blocks there are being wrapped all onto one line,
this changes the css so each line of the code block will be on its own.

### CODE REVIEW

File: assets/css/style.scss [hunk 1]
```diff
@@ -223,7 +223,7 @@ div.breadcrumbs.bootstrap .breadcrumb-item + .breadcrumb-item:hover::before {
 
 h2 a,
 .api code {
-      white-space: nowrap;
+      white-space: pre;
```
Comment (lsf37) [inline assets/css/style.scss:226]: I had always assumed that was there for a reason. @kent-mcleod, @wom-bat do you remember anything on that?

**Comment by axel-h** [comment]
Thanks. Seem to work for me in Chrome.

File: assets/css/style.scss (hunk 1 above)
Comment (ratmice) [reply assets/css/style.scss:226]: Looks like it was added ~4 years ago in 09707fd420b1996b96ebb722a58afba7c8c7976d by @WellMcGarnicle (though I certainly doubt I would remember any particulars)

Comment (kent-mcleod) [reply assets/css/style.scss:226]: This style is for API docs to stop code sections from wrapping. The style was guarded by an api class inherited by using the API layout.  At some point docs like GettingStarted started also using the API layout in order to get a table-of-contents on the side.  It seems like `pre` is an improvement though.

**Review by kent-mcleod — APPROVED** [review]

File: assets/css/style.scss (hunk 1 above)
Comment (kent-mcleod) [reply assets/css/style.scss:226]: (The original issue was that the invocation names on https://docs.sel4.systems/projects/sel4/api-doc.html would get broken up when the text wrapped on mobile screens I think).

Comment (lsf37) [reply assets/css/style.scss:226]: Ok, then I think `pre` is indeed an improvement.

**Review by lsf37 — APPROVED** [review]

**Pull request merged**